src/App.tsx

```diff
@@ -17,18 +61,39 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setFilteredBy={setFilteredBy}
+                setQuery={setQuery}
+                filteredBy={filteredBy}
+                query={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+
+              { isLoading
+                ? <Loader />
+                : (
+                  <TodoList
+                    todo={filteredTodos}
+                    selectedTodoId={selectedTodo}
+                    selectedTodo={setSelectedTodo}
+                    selectedUserId={setSelectedUserId}
+                  />
+                )}
             </div>
           </div>
         </div>
       </div>
-
-      <TodoModal />
+      {selectedTodo !== 0
```

Comment:
You could just check this way
```suggestion
      {selectedTodo
```